src/ood-gen/bin/watch_scrape.ml

```diff
@@ -1,76 +1,124 @@
 open Piaf
+open Lwt_result.Syntax
 
-type t = {
+type watch = {
   name : string;
   embedPath : string;
   thumbnailPath : string;
-  description : string;
-  year : string;
+  description : string option;
+  date : string;
```

Comment:
As suggested in the issue, an alternative would be to use both a `last_modified_date` and `created_date` fields